packages/seal-elasticsearch-adapter/ElasticsearchConnection.php

```diff
@@ -119,6 +120,8 @@ public function search(Search $search): Result
         foreach ($search->filters as $filter) {
             if ($filter instanceof IdentifierCondition) {
                 $query['ids']['values'][] = $filter->identifier;
+            } elseif ($filter instanceof SearchCondition) {
+                $query['query_string']['query'] = $filter->query;
```

Comment:
@wachterjohannes elasticsearch is a little bit confusing here not sure what the best case is for search that query.

Reply:
would keep it like that - i would say it is the "normal" search! but the better option would be `match` but there you have to set a specific field